tests/testapps/CTCatalog/CTCatalog/Tests/CTCFrameTestViewController.mm

```diff
@@ -27,6 +26,11 @@ @interface CTFrameLineTestView : UIView {
 @implementation CTFrameLineTestView {
 }
 
+- (void)setLineRef:(CTLineRef)lineRef {
```

Comment:
rather than doing this yourself, just declare the property as @property(nonatomic, retain)